docs/application/toc_all.md

```diff
@@ -72,6 +77,89 @@
 #### [Data Control](/application/dotnet/guides/app-management/data-control.md)
 
+###  Connectivity and Wireless
+#### [Overview](/application/dotnet/guides/connectivity/overview.md)
+#### [Connection Management](/application/dotnet/guides/connectivity/connection.md)
+#### [Network Service Discovery](/application/dotnet/guides/connectivity/nsd.md)
+#### [Wi-Fi](/application/dotnet/guides/connectivity/wifi.md)
+#### [Wi-Fi Direct®](/application/dotnet/guides/connectivity/wifi-direct.md)
+#### [Bluetooth](/application/dotnet/guides/connectivity/bluetooth.md)
+#### [NFC](/application/dotnet/guides/connectivity/nfc.md)
+#### [Smart Card](/application/dotnet/guides/connectivity/smartcard.md)
+#### [Smart Traffic Control (STC)](/application/dotnet/guides/connectivity/stc.md)
+#### [IoT Connectivity](/application/dotnet/guides/connectivity/iotcon.md)
+
+### Data
+#### [Overview](/application/dotnet/guides/data/overview.md)
+#### [Data Storages](/application/dotnet/guides/data/data-storages.md)
+
+### Internationalization
+#### [Overview](/application/dotnet/guides/internationalization/overview.md)
+#### [Localization](/application/dotnet/guides/internationalization/localization.md)
+
+### Libraries
+#### OpenTK
+##### [Overview](/application/dotnet/guides/libraries/opentk/index.md)
+##### [Quickstart](/application/dotnet/guides/libraries/opentk/quickstart.md)
+##### [Window of OpenTK](/application/dotnet/guides/libraries/opentk/window-of-opentk.md)
+#### [SkiaSharp](/application/dotnet/guides/libraries/skiasharp/index.md)
+#### Wearable Circular UI
+##### [Overview](/application/dotnet/guides/libraries/wcircularui/index.md)
+##### [Quickstart](/application/dotnet/guides/libraries/wcircularui/quickstart.md)
+##### [Tizen Wearable XAML App Template](/application/dotnet/guides/libraries/wcircularui/quickstart_tizenxamlapptemplate.md)
+#### [Third Party Libraries for Tizen .NET Application](/application/dotnet/guides/libraries/thirdparty/index.md)
+
+### Location
+#### [Overview](/application/dotnet/guides/location/overview.md)
+#### [Location Information](/application/dotnet/guides/location/location.md)
+#### [Geofences](/application/dotnet/guides/location/geofences.md)
+
+### Machine Learning
+#### [Overview](/application/dotnet/guides/machine-learning/overview.md)
+#### [Single Shot](/application/dotnet/guides/machine-learning/singleshot.md)
+#### [Pipeline](/application/dotnet/guides/machine-learning/pipeline.md)
+
+### Maps
+#### [Overview](/application/dotnet/guides/maps/maps.md)
+#### [Getting HERE Maps Credentials](/application/dotnet/guides/maps/here-credentials.md)
+
+### Media and Camera
+#### [Overview](/application/dotnet/guides/multimedia/overview.md)
+#### [Media Content](/application/dotnet/guides/multimedia/media-content.md)
+#### [Metadata](/application/dotnet/guides/multimedia/metadata.md)
+#### [Thumbnail Images](/application/dotnet/guides/multimedia/thumbnail-images.md)
```

Comment:
Hello, @annie-samsung 

Would you please guide @sagarkedari, about the md file structure and the URL relation?

@sagarkedari, md files structure becomes URL of each page. 
As you can see that the links in left navigation bar are working well, there is no need to use same folder name & menu name.

If we use same folder name to the LNB menu, it would be hard to understand what that section is about.
Likewise that, if you use exact menu name to the folder, the folder path would be too long, which will results to too long URL.

Reply:
@mijin85cho sure will keep that in mind. We have identified the MD files and did not find any content change from 1387 to 1483. For 1387 we used the list Nawab prepared. I will be doing some random grep checks by Monday or Tuesday we shall be posting our final feedback. We are now checking images.